app/views/project/editor/editor.pug

```diff
@@ -37,6 +37,7 @@ div.full-size(
 			keybindings="settings.mode",
 			font-size="settings.fontSize",
 			auto-complete="settings.autoComplete",
+			enable-auto-complete-features="true",
```

Comment:
We could only enable this only for users with permission to write, and likewise with the permissions on the server side routes. Not really essential, but reduce load slightly, and particularly in the case of a large number of read-only views of a project if it was public say.